eng/Version.Details.xml

```diff
@@ -10,197 +10,197 @@
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
     <!--
          Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
          All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20575.3">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20580.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
+      <Sha>c8a97ff77957202323b64b8513f363366010343f</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NETCore.App.Internal" Version="6.0.0-alpha.1.20575.3">
```

Comment:
@pranavkm @mmitche this didn't get updated again. @pranavkm updated it manually last time and it worked.
https://github.com/dotnet/extensions/pull/3703#discussion_r530588130

I'm seeing build failures like:
```
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '6.0.0-alpha.1.20580.2' was not found.
  - The following frameworks were found:
      5.0.0 at [F:\workspace\_work\1\s\.dotnet\shared\Microsoft.NETCore.App]
      6.0.0-alpha.1.20575.3 at [F:\workspace\_work\1\s\.dotnet\shared\Microsoft.NETCore.App]
```

Reply:
@Tratcher This package is no longer produced, as of mid-november.
/cc @jkoritzinsky

Reply:
@mmitche it looks like this is talking about the FrameworkReference, not the no-longer-produced package.

Reply:
Yes, this needs to be switched to another non-shipping package from dotnet/runtime. The extensions transport package should work?